### ISSUE TITLE
Need Support for .webm Audio file.[Feature Request] 

### ISSUE THREAD

**Issue by adhitht123** [telegramdesktop/tdesktop#10437]
### Is your feature request related to a problem?
No.

### Describe the solution you'd like
I need to play .webm audio file on native telegram audio player. Now currently it opens an external player. There are also other unsupported audio files Please try to add all these to telegram.

**Comment by ell1e** [comment]
Yes, this would be useful, also maybe someone should reopen #2842. mp4 just has some licensing issues as website owner, and webm audio+video won't have previews in telegram while it works in discord. would be cool to see it fixed.

**Comment by ell1e** [comment]
webm still should happen

**Comment by ell1e** [comment]
:robot: :vomiting_face:
